lib/bumper.js

```diff
@@ -50,40 +56,48 @@ class Bumper {
       return Promise.resolve()
     }
 
-    return this._getMergedPrInfo()
+    return this.vcs.getLastCommitMsg()
+      .then((commitMessage) => {
+        if (commitMessage === 'Automated version bump') {
+          throw new Cancel('Skipping bump on version bump commit.')
+        }
+        return Promise.resolve()
```

Comment:
I'm not positive. I know if you return a primitive value (not a promise) it ends up returning a promise that resolves as that value, but I'm not sure about `undefined`, it's probably not necessary. I was chasing down an issue where something didn't have a `.then()` so I was being extra cautious. 

You're probably right though, I don't think it's needed.